### ISSUE TITLE
docker.service.list() empty array

### ISSUE THREAD

**Issue by kanedauy** [AgustinCB/docker-api#19]
when I do this I get an empty array
I am using this the correct way?

```javascript
const Docker = require('node-docker-api').Docker
const docker = new Docker({
  socketPath: '/var/run/docker.sock'
})

 // GET: /services
 function serviceList(req, reply) {
   docker.service.list().then((services) => {
     console.dir(services)
     reply.sendStatus(200)
   }).catch((err) => {
     sendError(err, reply)
   })
 }
```

**Comment by AgustinCB** [comment]
Yeah, your code looks correct to me.

Can you show me the output of these commands, please?

`docker service ls`
`docker info`

Thanks!

**Comment by kanedauy** [comment]
```
$ docker service ls
ID            NAME            MODE        REPLICAS  IMAGE
7vro2ievs1x0  redis           replicated  1/1       redis:latest
iw1p5a81wndq  swarm-listener  replicated  1/1       vfarcic/docker-flow-swarm-listener:latest
mjgsutem0jj5  proxy           replicated  1/1       vfarcic/docker-flow-proxy:latest
w4aj4w1utmjo  mongodb1        replicated  1/1       mongo:latest


$ docker info
Containers: 2
 Running: 2
 Paused: 0
 Stopped: 0
Images: 16
Server Version: 17.03.0-ce
Storage Driver: aufs
 Root Dir: /var/lib/docker/aufs
 Backing Filesystem: extfs
 Dirs: 87
 Dirperm1 Supported: true
Logging Driver: json-file
Cgroup Driver: cgroupfs
Plugins:
 Volume: local
 Network: bridge host macvlan null overlay
Swarm: active
 NodeID: k660686o41hbweyctfawyuvfd
 Is Manager: true
 ClusterID: b926l3p5wktrjff60edzxjh43
 Managers: 1
 Nodes: 2
 Orchestration:
  Task History Retention Limit: 5
 Raft:
  Snapshot Interval: 10000
  Number of Old Snapshots to Retain: 0
  Heartbeat Tick: 1
  Election Tick: 3
 Dispatcher:
  Heartbeat Period: 5 seconds
 CA Configuration:
  Expiry Duration: 3 months
 Node Address: ###.###.###.###
 Manager Addresses:
  ###.###.###.###:2377
Runtimes: runc
Default Runtime: runc
Init Binary: docker-init
containerd version: 977c511eda0925a723debdc94d09459af49d082a
runc version: a01dafd48bc1c7cc12bdb01206f9fea7dd6feb70
init version: 949e6fa
Security Options:
 apparmor
 seccomp
  Profile: default
Kernel Version: 4.4.0-66-generic
Operating System: Ubuntu 16.04.2 LTS
OSType: linux
Architecture: x86_64
CPUs: 1
Total Memory: 992.4 MiB
Name: cloud
ID: X7O2:5YK3:JTO6:SAQS:MIKI:3BW3:GZLR:XGBP:6BJH:MTJ7:3OH4:MDTQ
Docker Root Dir: /var/lib/docker
Debug Mode (client): false
Debug Mode (server): false
Username: asuero
Registry: https://index.docker.io/v1/
Experimental: false
Insecure Registries:
 127.0.0.0/8
Live Restore Enabled: false
```

**Comment by AgustinCB** [comment]
I think you just found a bug :). I'll work on a fix and let you know. I hope to have something tomorrow.

Thank you!

**Comment by kanedauy** [comment]
Thanks a lot!

**Comment by AgustinCB** [comment]
Nice catch. It should be fixed on version 1.1.9 with this commit: https://github.com/AgustinCB/docker-api/commit/a8b3056e71c3219fac96996e44b077353b7b3cbc. I also modified the test to ensure it's fixed.
